ThirdPartyNotices.txt

```diff
@@ -5993,3 +5993,32 @@ https://github.com/tensorflow/tfjs
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+——
+
+curl/curl
```

Comment:
Please note this library uses a non-standard license type that might need extra internal reviews.

Reply:
We are consuming the curl from vcpkg, not from git directly - would vcpkg licence type override the one seeing here?